addons/binding/org.openhab.binding.nuki/README.md

```diff
@@ -0,0 +1,134 @@
+# Nuki Binding
+
```

Comment:
Throughout the README, you often have 3 empty lines - please reduce them to a single one.